keyboards/mokey/mokey12x2/mokey12x2.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2021 Yiancar-Designs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#if defined(ENCODER_ENABLE)
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) {
```

Comment:
Please pay more attention when applying these suggestions, you were only asked to change one line and have reintroduced the original issue in a worse way....
```suggestion
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) {
```

Reply:
Sorry

Reply:
Have been changed